app/main.py

```diff
@@ -1,8 +1,59 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        comfort_class: int,
+        clean_mark: int,
+        brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: float,
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int,
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> int:
+        total_income = 0.0
+
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                price = self.calculate_washing_price(car)
+                total_income += price
+                self.wash_single_car(car)
+
+        return round(total_income, 1)
```

Comment:
The `serve_cars` method initializes `total_income` as a float (0.0) but returns it as an integer using `round()`. Consider initializing `total_income` as an integer (0) if you intend to return an integer value, or ensure the return type is consistently a float if that is the intended behavior.